### PR TITLE
Add non-interactive mode to installLisk.sh

### PR DESCRIPTION
**Pull request by Nazgolze**

### CODE REVIEW

File: downloaded/installLisk.sh [hunk 1]
```diff
@@ -39,6 +39,10 @@ if [ "$USER" == "root" ]; then
 fi
 
 prereq_checks() {
+	if [[ $- != *i* ]]; then
```
Comment (fchavant) [inline downloaded/installLisk.sh]: Shouldn't this be an option that can be passed?

Comment (Nazgolze) [reply downloaded/installLisk.sh]: We'll use the [-n] that I forgot to get rid of in the usage output ;-)

**Comment by fchavant** [comment]
To clarify:
 - this PR removes usages of `tput` so that running `installLisk.sh` in a non-interactive shell does not fail
 - non interactive mode can be achieved by passing all required parameter (no change or new flag required)
@Nazgolze correct?

**Comment by Nazgolze** [comment]
@fchavant correct

**Pull request merged**